forgettable-frontend/src/components/Dialog/Dialog.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import classes from './Dialog.module.css';
+
+function Dialog(props) {
```

Comment:
Can we please change 'Dialog' to 'Modal' as it will be consistent with the issue.